src/ts/roam/roam.ts

```diff
@@ -77,6 +84,20 @@ export const Roam = {
         this.applyToCurrent(node => node.withCursorAtTheEnd())
     },
 
+    async replace(element: HTMLElement, mod: { (orig: string): string }) {
```

Comment:
so this is basically apply but for HtmlElement instead of current block, I feel like they should follow similar interface (i.e. in terms of RoamNode)